oauth2_client/urls.py

```diff
@@ -0,0 +1,18 @@
+from importlib import import_module
```

Comment:
add doctring

Reply:
What would you like that to say? Something like this? 
"""
oauth2_client main URL configuration file

The `urlpatterns` list routes URLs to views. For more information please see:
    https://docs.djangoproject.com/en/1.11/topics/http/urls/
"""

This is what every django tutorial explains in the first 5 minutes. I can add that, but I don't see any value in a comment like that.